### ISSUE TITLE
Multible domains in proxy.conf?

### ISSUE THREAD

**Issue by KristjanVall** [DanielDent/docker-nginx-ssl-proxy#5]
How can i conf proxy.conf for multy fomains?

**Comment by xiaods** [comment]
not yet support.

**Comment by benrowe** [comment]
Is it possible to eject the `/etc/nginx/conf.d/default.conf` once it's been created and provide our own?

**Comment by DanielDent** [comment]
Yes - see https://github.com/DanielDent/docker-nginx-ssl-proxy#customizing
